Python-requests stub

evilon opened the pull request:
This stub uses the python requests -library for TLS connection. 
- [x] Check why one localhost check fails
  - it failed because the ca cert was provided using wrong parameter.

evilon committed:
Initial version of python-requests

oherrala commented on stubs/python-requests/run.py:
```diff
@@ -0,0 +1,15 @@
+cafile = sys.argv[3] if len(sys.argv) > 3 else False
+
+try:
+    r = requests.get("https://" + host + ":" + port, cert=cafile)
```
If I'm looking the correct [docs](http://docs.python-requests.org/en/master/user/advanced/) for `requests`, the `cert` parameter is used for TLS's client certificate authentication. CA bundle is given with [`verify`](http://docs.python-requests.org/en/master/user/advanced/#ssl-cert-verification) parameter.

evilon committed:
python-requests - use verify argument instead of cert

According to the documentation, "cert" parameter is used for client auth.
http://docs.python-requests.org/en/master/user/advanced/

Thanks @oherrala for spotting this.

oherrala commented:
👍 LGTM

``` sh
$ python run.py google.com 443
VERIFY SUCCESS
$ python run.py google.com 443 certs.pem
VERIFY FAILURE
$ python run.py google.com 443 ~/Downloads/certs.pem 
VERIFY SUCCESS
```

jviide commented on stubs/python-requests/run.py:
```diff
@@ -0,0 +1,15 @@
+try:
+    r = requests.get("https://" + host + ":" + port, verify=cafile)
+except requests.exceptions.SSLError as err:
+    #    print err
```
The `# print err` line could be removed.

jviide commented on stubs/python-requests/README.md:
````diff
@@ -0,0 +1,23 @@
+
+ # Examples
+
+ ```
````
Maybe ```sh?

evilon committed:
python-requests - formatting and other cleanup

jviide commented:
LGTM 👍
